source/deprecations/v1.x.html.md

```diff
@@ -704,9 +704,17 @@ dropdown options, the following code could be used:
 import Ember from "ember";
 
 export default Ember.Component.extend({
-  content: [],
+  content: null,
   selectedValue: null,
 
+  didInitAttrs(attrs) {
+    var content = this.get('content');
+    
+    if (!content) {
+      this.set('content', []);
```

Comment:
this also then wouldn't work correctly with the proposed ES7 property initializers.